src/main/java/com/commercetools/paymenttoorderprocessor/jobs/actions/MessageFilter.java

```diff
@@ -53,14 +53,7 @@ public CartAndMessage process(PaymentTransactionCreatedOrUpdatedMessage message)
                 if (oCart.isPresent()) {
                     final Cart cart = oCart.get();
                     if (cart.getCartState() != CartState.ORDERED) {
-                        if (isCartAmountEqualToTransaction(cart, payment, message)) {
-                            return new CartAndMessage(cart, message);
-                        } else {
-                            LOG.error("Cannot create Order for Cart {}. "
-                                       + "The transaction amout of Transaction {} does not match cart.",
-                                        cart.getId(), message.getTransactionId());
-                            messageProcessedManager.setMessageIsProcessed(message);
-                        }
+                        return new CartAndMessage(cart, message);
                     } else {
```

Comment:
I think that kind of breaking change, because a "cart-check" was removed. Maybe we have to add this to a change-log, which can be created within the readme,wdyt?

Reply:
You're right, it is a breaking a change. We write these things in the Release notes, which I will do once I am about to release later. Why do you think we should create a CHANGELOG in README file?

Reply:
ok, adding it to the release not is also fine...